src/pathfinding_service/service.py

```diff
@@ -98,6 +98,8 @@ def __init__(  # pylint: disable=too-many-arguments
             message_received_callback=self.handle_message,
             address_reachability_changed_callback=self.handle_reachability_change,
         )
+        self.matrix_listener.startup_finished.wait(timeout=1)
```

Comment:
Any opinion on what is a reasonable timeout for this?

Reply:
As long as we don't get a timeout it's high enough and since it is only a timeout and not a fixed wait, it doesn't hurt to set it to a few more seconds. Everything between 1 and 10 seconds seems reasonable.

Reply:
Fixed.